specs/experimental/custom-gas-token.md

```diff
@@ -141,6 +141,22 @@ flowchart LR
 
 ## Contract Modifications
 
+### Interface
```

Comment:
I think @mds1 suggested that we call this interface `IGasToken`, we could make this `## IGasToken Interface`

and add a little description about the interface, like why it exists (encapsulates shared interface)

We should expand on the arguments + return types of the interface, it could be good just to do a solidity block and define the interface itself. We want to describe what each function in the interface does

Reply:
should we add `OptimismPortal` to the contracts that use the interface? it uses `gasPayingToken()` but no the other methods. still, it might be useful to just have this interface for all contracts affected by the custom gas token

Reply:
(otherwise, I already implemented the changes)